app/src/main/java/com/imashnake/animite/ui/elements/home/MediaSmall.kt

```diff
@@ -34,8 +34,7 @@ fun MediaSmall(image: String?, anime: String?) {
         onClick = { },
         modifier = Modifier
             .wrapContentHeight()
-            .width((115 + 12 + 12).dp)
-            .padding(start = 12.dp, end = 12.dp),
+            .width((115 + 12 + 12).dp),
```

Comment:
```suggestion
            .width(115.dp),
```
Note that the width should be `115.dp`. On second thought, however, I might make the width dynamic so that the *entire* image is displayed. This change will be applied later, for now, I think we should stick to the mock-up.